### PR TITLE
style: adjust line-height usage after change from px value to unitless

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fixup to  https://github.com/nextcloud/server/pull/46876

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team

### CODE REVIEW

File: css/talk-admin-settings.css [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 .settings-section-placeholder {
 	--settings-section-placeholder-header-height: 30px;
-	--settings-section-placeholder-line-height: var(--default-line-height);
+	--settings-section-placeholder-line-height: calc(var(--default-line-height) * var(--default-font-size));
```
Comment (ShGKme) [inline css/talk-admin-settings.css:8]:
```suggestion
	--settings-section-placeholder-line-height: 1lh;
```

File: src/components/NewMessage/NewMessage.vue [hunk 2]
```diff
@@ -1067,7 +1067,7 @@ export default {
 
 	// Override NcRichContenteditable styles
 	:deep(.rich-contenteditable__input) {
-		--contenteditable-space: calc((var(--default-clickable-area) - var(--default-line-height) - 4px) / 2);
+		--contenteditable-space: calc((var(--default-clickable-area) - var(--default-line-height) * var(--default-font-size) - 4px) / 2);
```
Comment (ShGKme) [inline src/components/NewMessage/NewMessage.vue:1070]:
```suggestion
		--contenteditable-space: calc((var(--default-clickable-area) - 1lh - 4px) / 2);
```

**Review by ShGKme — COMMENTED** [review]
To not rely on unit kind

File: css/talk-admin-settings.css (hunk 1 above)
Comment (Antreesy) [reply css/talk-admin-settings.css:8]: Dark magic indeed =)

**Comment by ShGKme** [comment]
`lh` is not supported by Firefox 🙈

**Comment by ShGKme** [comment]
> `lh` is not supported by Firefox 🙈

`lh` function is not supported, but `lh` unit is 🙈

Not a great support, but _almost_ fine.

Let's go with `lh` then.

https://caniuse.com/mdn-css_types_length_lh

**Review by ShGKme — APPROVED** [review]

**Pull request merged**